index.html

```diff
@@ -801,7 +803,8 @@
    // event handlers
    var mouseClick = function(e)
    {
-       var cue_ball = ballList.balls[0];
+       //Obsolete now but I leave it here for easy modifications
+      /* var cue_ball = ballList.balls[0];
```

Comment:
@ENiGMA9 if this isn't needed feel free to clean up.  Github keeps the history nicely :)